http-kernel-fixtures/sub-folder/cookie_page4.php

```diff
@@ -0,0 +1,21 @@
+<?php
+    $resp = new Symfony\Component\HttpFoundation\Response();
+    if (method_exists('Symfony\Component\HttpFoundation\Cookie', 'create')) {
+        $cook = Symfony\Component\HttpFoundation\Cookie::create('srvr_cookie', 'srv_var_is_set', 0, '/sub-folder/');
```

Comment:
please use full names for the `$cookie` variable rather than an abbreviated name (for `$resp`, this is enforced by the FixturesKernel so we cannot change it here)

Reply:
Fixed. Note: This was a copy/paste of cookie_page1.